spatialpandas/io/__init__.py

```diff
@@ -1,3 +1,3 @@
 from .parquet import (  # noqa
     read_parquet, read_parquet_dask, to_parquet, to_parquet_dask
-)
+)
```

Comment:
Fix missing final newline:
```suggestion
)

```